src/tcti-tabrmd.c

```diff
@@ -522,3 +522,25 @@ tss2_tcti_tabrmd_init (TSS2_TCTI_CONTEXT *context,
                                        TCTI_TABRMD_DBUS_TYPE_DEFAULT,
                                        TCTI_TABRMD_DBUS_NAME_DEFAULT);
 }
+
+static TSS2_RC
+tss2_tcti_tabrmd_init_config (TSS2_TCTI_CONTEXT *context,
+                              size_t            *size,
+                              const char        *conf) {
```

Comment:
I could use G_GNUC_UNUSED from glib, not sure what your preference is: https://developer.gnome.org/glib/stable/glib-Miscellaneous-Macros.html#G-GNUC-UNUSED:CAPS

(void) is compiler independent.